packages/components/src/toggle-group-control/test/index.tsx

```diff
@@ -121,7 +124,7 @@ describe( 'ToggleGroupControl', () => {
 			'Click for Sumptuous Caponata'
 		);
 
-		fireEvent.focus( secondRadio );
+		await user.click( secondRadio );
```

Comment:
Did we mean to change the focus to be click instead?

Reply:
In both tests, we're testing whether a tooltip appears or not over a radio form item when the item is focused. If `userEvent` has some method for simulating "move the mouse over the input", that would be great to use here. The tooltip is then supposed to appear.

`user.tab()` works because it focuses the right element, but it's quite an indirect and non-obvious way to do that.

I think `user.click` is wrong. The tooltip indeed doesn't appear, so the test passes, but it doesn't appear for the wrong reasons.

Reply:
I agree with Jarda on this. Also IMHO the safest bet is to just keep triggering the focus event, but just not with `fireEvent`:

```suggestion
		secondRadio.focus();
```

Reply:
Thanks for the feedback! I agree on keeping it simple in this case. Does https://github.com/WordPress/gutenberg/pull/45627/commits/848e112a1007b83794893e676c78a48a938619b9 work?